modules/openfast-library/src/FAST_Library.f90

```diff
@@ -318,6 +318,10 @@ subroutine FAST_Update(iTurb, NumInputs_c, NumOutputs_c, InputAry, OutputAry, En
    OutputAry(1)              = Turbine(iTurb)%m_FAST%t_global 
    OutputAry(2:NumOutputs_c) = Outputs 
 
+   CALL FillOutputAry_T(Turbine(iTurb), Outputs)
```

Comment:
This is duplicated from the lines above. Does this need to be here or is it a mistake?

Reply:
I am not sure.  I'll need to ask Ganesh--he did this part.

Reply:
@gantech see these comments

Reply:
I removed the repeated lines and reran a case and got identical results, so I have removed the duplicates.